packages/@webex/plugin-meetings/src/meeting/voicea-meeting.ts

```diff
@@ -0,0 +1,187 @@
+      timestamp: data.timestamp,
+      speaker,
+    };
+    transcriptData.captions.push(captionData);
+  }
+  const {transcripts = []} = data;
+  const transcriptsPerCsis = new Map();
+
+  for (const transcript of transcripts) {
+    const {
+      text,
+      transcript_language_code: currentSpokenLanguage,
+      csis: [csisMember],
+    } = transcript;
+
+    const newCaption = `${transcriptsPerCsis.get(csisMember)?.text ?? ''} ${text}`;
+
+    // eslint-disable-next-line camelcase
+    transcriptsPerCsis.set(csisMember, {text: newCaption, currentSpokenLanguage});
+  }
+  const fakeTranscriptionIds = [];
+
+  for (const [key, value] of transcriptsPerCsis) {
+    const {needsCaching, speaker} = getSpeakerFromProxyOrStore({
+      meetingMembers: meeting.members.membersCollection.members,
+      transcriptData,
+      csisKey: key,
+    });
+
+    if (needsCaching) {
+      transcriptData.speakerProxy[key] = speaker;
+    }
+    const {speakerId} = speaker;
+    const fakeId = `${transcriptId}_${speakerId}`;
+    const captionData = {
+      id: fakeId,
+      isFinal: data.isFinal,
+      translations: value.translations,
+      text: value.text,
+      currentCaptionLanguage: value.currentSpokenLanguage,
+      timestamp: value?.timestamp,
+      speaker,
+    };
+
+    const fakeTranscriptIndex = transcriptData.captions.findIndex(
+      (transcript) => transcript.id === fakeId
+    );
+
+    if (fakeTranscriptIndex !== -1) {
+      transcriptData.captions.splice(fakeTranscriptIndex, 1);
+    }
+
+    fakeTranscriptionIds.push(fakeId);
+    transcriptData.captions.push(captionData);
+  }
+  transcriptData.interimCaptions[transcriptId] = fakeTranscriptionIds;
+};
+
+export const processHighlightCreated = ({data, meeting}) => {
+  const transcriptData = meeting.transcription;
+
+  if (!transcriptData.highlights) {
+    transcriptData.highlights = [];
+  }
+
+  const csisKey = data.csis && data.csis.length > 0 ? data.csis[0] : undefined;
+  const {needsCaching, speaker} = getSpeakerFromProxyOrStore({
+    meetingMembers: meeting.members.membersCollection.members,
+    transcriptData,
+    csisKey,
+  });
+
+  if (needsCaching) {
+    transcriptData.speakerProxy[csisKey] = speaker;
+  }
+
+  const highlightCreated = {
+    id: data.highlightId,
+    meta: {
+      label: data.highlightLabel,
+      source: data.highlightSource,
+    },
+    text: data.text,
+    timestamp: data.timestamp,
+    speaker,
+  };
+
+  voicea.highlights.push(highlightCreated);
+};
+
+export const EVENT_TRIGGERS = {
+  VOICEA_ANNOUNCEMENT: 'voicea:announcement',
+  CAPTION_LANGUAGE_UPDATE: 'voicea:captionLanguageUpdate',
+  SPOKEN_LANGUAGE_UPDATE: 'voicea:spokenLanguageUpdate',
+  CAPTIONS_TURNED_ON: 'voicea:captionOn',
+  TRANSCRIBING_ON: 'voicea:transcribingOn',
+  TRANSCRIBING_OFF: 'voicea:transcribingOff',
+
+  NEW_CAPTION: 'voicea:newCaption',
+  EVA_COMMAND: 'voicea:wxa',
+  HIGHLIGHT_CREATED: 'voicea:highlightCreated',
+};
```

Comment:
Should add it along with other meeting constants

Reply:
Turns out it was being used from the Voicea plugin directly. So removed it from here.